R/app_run.R

```diff
@@ -536,6 +539,10 @@ app_run <- function(nhanes_data = cardioStatsUSA::nhanes_data,
  # Server ------------------------------------------------------------------
  server <- function(input, output, session) {
 
+  # initiate hints on startup with custom button and event
+  hintjs(session, options = list("hintButtonLabel"="Hope this hint was helpful"),
+         events = list("onhintclose"=I('alert("Wasn\'t that hint helpful")')))
```

Comment:
can you remove the events argument? I don't think we need any events on close.

Reply:
can you remove the options argument as well?

basically just change this to `hintjs(session)`